app/scripts/modules/core/pipeline/config/stages/webhook/webhookStage.html

```diff
@@ -0,0 +1,97 @@
+<div class="form-horizontal">
+  <stage-config-field label="Your service's URL">
```

Comment:
I don't know if we have any fields that are prefixed with "Your". Maybe just "Payload URL" would be better here?